types/passport-openidconnect/index.d.ts

```diff
@@ -0,0 +1,328 @@
+         * @param appState - additional app state to be stored in session
+         * @param meta - metadata of the request
+         * @param cb - {@link SessionStoreCallback} to execute after storing session
+         */
+        store(
+            req: Request,
+            ctx: SessionStoreContext,
+            appState: object | undefined,
+            meta: object | undefined,
+            cb: SessionStoreCallback,
+        ): void;
+
+        /**
+         * Verify request state.
+         *
+         * This implementation simply compares the state parameter in the request to the
+         * value generated earlier and stored in the session.
+         *
+         * @param req - Request object of the incoming http request
+         * @param reqState - Generated uuid string that identifies a unique auth session across multiple requests to OIDC provider
+         * @param cb - {@link SessionVerifyCallback} to execute after storing session
+         */
+        verify(req: Request, reqState: string, cb: SessionVerifyCallback): void;
+    }
+
+    /**
+     * Options available to pass into {@link OpenIDConnectStrategy} during instantiation.
+     * Majority of these parameters adhere to the OAuth2 Auth Code Flow spec.
+     *
+     * @see https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth
+     */
+    interface StrategyOptions {
+        issuer: string;
+        authorizationURL: string;
+        tokenURL: string;
+        callbackURL: string;
+        userInfoURL: string;
+        clientID: string;
+        clientSecret: string;
+
+        acrValues?: string | undefined;
+        agent?: boolean | Agent | undefined;
+        claims?: object | undefined;
+        customHeaders?: OutgoingHttpHeaders | undefined;
+        display?: string | undefined;
+        idTokenHint?: string | undefined;
+        loginHint?: string | undefined;
+        maxAge?: string | undefined;
+        nonce?: string | undefined;
+        prompt?: string | undefined;
+        proxy?: boolean | undefined;
+        responseMode?: string | undefined;
+        scope?: string | string[] | undefined;
+        uiLocales?: string | undefined;
+
+        /**
+         * If defined, the {@link express.Request | Request} object will be passed into {@link VerifyFunction}
+         */
+        passReqToCallback?: boolean | undefined;
+        /**
+         * Unique session identifier for this particular provider.
+         * If none is given, the provider's hostname will be used.
+         */
+        sessionKey?: string | undefined;
+        /**
+         * Session store instance with interface compliant to {@link SessionStore}
+         */
+        store?: SessionStore | undefined;
+        /**
+         * If defined, skips the loading of the user profile
+         */
+        skipUserProfile?: boolean | undefined;
+    }
+
+    type VerifyCallback = (err?: Error | null, user?: Express.User, info?: any) => void;
```

Comment:
hello 👋 i was just trying to use the `@types/passport-openidconnect` package and i'm noticing the following error when i try to build my package:

```bash
node_modules/@types/passport-openidconnect/index.d.ts:174:63 - error TS2694: Namespace 'global.Express' has no exported member 'User'.

174     type VerifyCallback = (err?: Error | null, user?: Express.User, info?: any) => void;
```

So I wondering if `Express.User` here is intentional? Or should it be something like:

```typescript
import { Request, User } from 'express';

...

 type VerifyCallback = (err?: Error | null, user?: User, info?: any) => void;
```